pages/nft/url/index.vue

```diff
@@ -287,8 +286,11 @@ export default class FetchIndex extends Vue {
       publisher,
       type = 'CreativeWork',
     } = this.iscnData
-    let { description = '' } = this.iscnData
+    let { description = '', keywords = '' } = this.iscnData
```

Comment:
Will this duplicate with https://github.com/likecoin/app-like-co/pull/486/files?
Should we check if `NewsPress` already exists?

Reply:
[🐛 Prevent duplicate keywords](https://github.com/likecoin/app-like-co/pull/488/commits/f5ac429246948669482dbcf222c48ac9ef597e74)